types/src/chainspec/vm_config/messages_limits.rs

```diff
@@ -0,0 +1,113 @@
+#[cfg(feature = "datasize")]
+use datasize::DataSize;
+use rand::{distributions::Standard, prelude::*, Rng};
+use serde::{Deserialize, Serialize};
+
+use crate::bytesrepr::{self, FromBytes, ToBytes};
+
+/// Configuration for messages limits.
+#[derive(Clone, Copy, PartialEq, Eq, Serialize, Deserialize, Debug)]
+#[cfg_attr(feature = "datasize", derive(DataSize))]
+#[serde(deny_unknown_fields)]
+pub struct MessagesLimits {
```

Comment:
**A nit:** Consider renaming to `MessageLimits`, per analogy to `OpcodeCosts` vs `OpcodesCosts`, `HostFunctionCosts` vs `HostFunctionsCosts`, etc.

Reply:
Renamed here: https://github.com/casper-network/casper-node/commit/3190647ec0012448907208e3a8406244433c5486